### PR TITLE
Added support for Python and MATLAB code

### LINKED ISSUE #14: Add support for Python, MATLAB, Julia and other scientific / mathematical languages
Python in particular. It's the most popularly used language according to PYPL.

### PR DESCRIPTION
**Pull request by Calman102**
Fixes #14

### CODE REVIEW

**Comment by bryansray** [comment]
Would love to see this merged down. Thanks for the effort, @Calman102 💯

**Comment by bobzomer** [comment]
Would be great to see this merged! Any idea on when we could see this?

**Comment by michael-hawker** [comment]
@bryansray @bobzomer were you able to pull this change down and see it working? If so, can you let us know. Thanks!

**Comment by bobzomer** [comment]
> @bryansray @bobzomer were you able to pull this change down and see it working? If so, can you let us know. Thanks!

Yes. I've tested the proposed change and it works.

File: ColorCode.Core/Compilation/Languages/MatLab.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.  All rights reserved.
```
Comment (michael-hawker) [inline ColorCode.Core/Compilation/Languages/MatLab.cs]:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

File: ColorCode.Core/Compilation/Languages/Python.cs [hunk 2]
```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.  All rights reserved.
```
Comment (michael-hawker) [inline ColorCode.Core/Compilation/Languages/Python.cs]:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

**Review by michael-hawker — COMMENTED** [review]
Updating file headers to match repo licensing now (see #20). Contributor has already signed .NET Foundation CLA, so should be no legal impact here. This is just a text change to align everything to the current proper state.

**Review by michael-hawker — APPROVED** [review]
Hopefully build should be good now and we can get this merged.

**Comment by michael-hawker** [comment]
Hmm, looks like something may have happened to our build since it last ran...

```
C:\hostedtoolcache\windows\dotnet\sdk\6.0.400\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(358,5): error NETSDK1127: The targeting pack Microsoft.NETCore.App is not installed. Please restore and try again. [D:\a\1\s\ColorCode.WinUI\ColorCode.WinUI.csproj]
```

@Arlodotexe mind investigating off of main?

We should be able to track the latest build here actually (and we knew it originally built as well): https://dev.azure.com/dotnet/CommunityToolkit/_build/results?buildId=79469&view=results

**Edit**: Yeah, same issue there... 🙁 I'll merge this since it built before and open a new issue to work towards a fix.

**Pull request merged**